CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -51,7 +51,7 @@ function Stop-ProcessIfExists {
     Invoke-Command -Session $Session -ScriptBlock {
         $Proc = Get-Process $Using:ProcessName -ErrorAction SilentlyContinue
         if ($Proc) {
-            $Proc | Stop-Process -Force
+            $Proc | Stop-Process -Force -PassThru | Wait-Process -ErrorAction SilentlyContinue
```

Comment:
It seems to be fine

Reply:
I wonder why do we have to add this SilentlyContinue ?

Reply:
If `Stop-Process` kills the process fast enough, `Wait-Process` could get dead process on input.